projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/order-detail-items.component.spec.ts

```diff
@@ -89,11 +107,21 @@ describe('OrderDetailItemsComponent', () => {
     };
 
     TestBed.configureTestingModule({
-      imports: [CardModule, I18nTestingModule],
+      imports: [CardModule, I18nTestingModule, FeaturesConfigModule],
       providers: [
         { provide: OrderDetailsService, useValue: mockOrderDetailsService },
+        {
+          provide: FeaturesConfig,
+          useValue: {
+            features: { level: "'1.1'", consignmentTracking: "'1.2'" },
```

Comment:
It probably should be like this `features: { level: '1.1', consignmentTracking: '1.2' },`